src/config/index.js

```diff
@@ -0,0 +1 @@
+export const BASE_URL = "http://localhost:5000";
```

Comment:
I would maybe put this is a API urls constants file, where we would later add other needed subroutes/url parts